executor/analyze_test.go

```diff
@@ -397,28 +428,31 @@ func checkHistogram(sc *stmtctx.StatementContext, hg *statistics.Histogram) (boo
 	return true, nil
 }
 
-func (s *testFastAnalyze) TestFastAnalyze(c *C) {
-	c.Skip("Skip this unstable test(#25782) and bring it back before 2021-07-29.")
+func TestFastAnalyze(t *testing.T) {
+	t.Parallel()
+	t.Skip("Skip this unstable test(#25782) and bring it back before 2021-07-29.")
+	store, clean := testkit.CreateMockStore(t)
+	defer clean()
 	var cls testutils.Cluster
 	store, err := mockstore.NewMockStore(
```

Comment:
You can remove this line.

Reply:
okay,you mean removing 434 and 435 ? I've removed just now.